pkgs/development/python-modules/fvcore/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv
+, lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pytestCheckHook
+, huggingface-hub
+, iopath
+, numpy
+, pillow
+, pyyaml
+, shapely
+, tabulate
+, termcolor
+, torch
+, torchvision
+, tqdm
+, yacs
+}:
+
+buildPythonPackage rec {
+  pname = "fvcore";
+  disabled = pythonOlder "3.6";
+  version = "0.1.5";
+
+  src = fetchFromGitHub {
+    owner = "facebookresearch";
+    repo = "fvcore";
+    rev = version;
```

Comment:
It seems like there isn't a tag or release with the specified version number. When attempting to build the package, the process fails with a 404 error. Here is the error message:
```
trying https://github.com/facebookresearch/fvcore/archive/0.1.5.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
^M  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0^M  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
^M  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0^M  0    14    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
curl: (22) The requested URL returned error: 404
error: cannot download source from any mirror
```